File: mask-python/tests/test_cross.py
import os
os.environ["MASK_ENCRYPTION_KEY"] = "my-test-secret-key-1234567890"

from mask_privacy.core.crypto import get_crypto_engine
engine = get_crypto_engine()

plaintext = "Hello from Python"
ciphertext = engine.encrypt(plaintext)
print(f"PYTHON ENCRYPTED: {ciphertext}")

File: mask-ts/tests/test_cross.ts
import { CryptoEngine } from '../src/core/crypto';
import * as process from 'process';

async function test() {
    process.env.MASK_ENCRYPTION_KEY = "my-test-secret-key-1234567890";
    const engine = await CryptoEngine.getInstanceAsync();
    
    const ciphertext = process.argv[2];
    console.log(`DECRYPTING IN TS: ${ciphertext}`);
    const plaintext = engine.decrypt(ciphertext);
    console.log(`TS RECOVERED: ${plaintext}`);
}

test().catch(console.error);
